src/components/local-scene-sidebar/local-scene-sidebar-styles.module.scss

```diff
@@ -5,33 +5,22 @@
 
 .container {
   @include backdropBlur();
-  @extend %verticalScrollBar;
   @include animationFunction($propertyToAnimate: all);
   border: $sidebar-border;
   border-radius: $sidebar-border-radius;
   position: absolute;
-  height: 95vh;
+  height: calc(95vh + #{$sidebar-cards-padding});
   width: $local-sidebar-width;
   top: $local-sidebar-top-margin;
   left: $site-gutter;
-  padding-bottom: $site-gutter;
+  padding: $site-gutter 0;
```

Comment:
I think we could add a margin on the top of the sidebar like on the design so the button has some space to the top of the page